app/controllers/discover.js

```diff
@@ -34,7 +34,7 @@ export default Controller.extend(Analytics, {
     },
     // TODO: Add a conversion from shareSource to provider names here if desired
     filterReplace: { // Map filter names for front-end display
-        'Open Science Framework': 'OSF',
+        'Open Science Framework': 'OSF Preprints',
         'Cognitive Sciences ePrint Archive': 'Cogprints',
         OSF: 'OSF Preprints',
```

Comment:
is this one still relevant?

Reply:
OSF preprints from share results has the provider/source as `OSF`.